packages/app/components/drop/drop-music.tsx

```diff
@@ -174,9 +174,9 @@ export const DropMusic = () => {
         if (isSaveDrop) {
           return baseSchema.concat(
             yup.object({
-              spotifyUrl: yup
-                .string()
-                .required("Spotify Song URL is required for saved drops."),
+              spotifyUrl: yup.string(),
+              appleMusicTrackId: yup.string(),
```

Comment:
// TODO - will be fixing this before merging. One of the two needs to be mandatory.